src/test/java/com/google/javascript/clutz/types_externs_with_platform.js

```diff
@@ -3,6 +3,7 @@ goog.provide('typesWithExterns.A');
 goog.provide('typesWithExterns.B');
```

Comment:
Could you split up `types_externs_with_platform.d.ts` into a part that's just the giant golden file and a part that has the bits we care about? This is going to be a maintenance nightmare otherwise.